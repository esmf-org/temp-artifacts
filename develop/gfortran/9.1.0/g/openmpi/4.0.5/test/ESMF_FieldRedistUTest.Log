2022-07-16 12:53:24
20220716 131042.552 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220716 131042.552 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220716 131042.552 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220716 131042.552 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220716 131042.552 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220716 131042.552 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220716 131042.552 INFO             PET0 Running with ESMF Version   : v8.4.0b04-2-gc831a7cd4a
20220716 131042.552 INFO             PET0 ESMF library build date/time: "Jul 16 2022" "12:50:27"
20220716 131042.552 INFO             PET0 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_9.1.0_openmpi_g_develop/esmf
20220716 131042.552 INFO             PET0 ESMF_COMM                   : openmpi
20220716 131042.552 INFO             PET0 ESMF_MOAB                   : enabled
20220716 131042.552 INFO             PET0 ESMF_LAPACK                 : enabled
20220716 131042.552 INFO             PET0 ESMF_NETCDF                 : enabled
20220716 131042.552 INFO             PET0 ESMF_PNETCDF                : disabled
20220716 131042.552 INFO             PET0 ESMF_PIO                    : enabled
20220716 131042.552 INFO             PET0 ESMF_YAMLCPP                : enabled
20220716 131042.554 INFO             PET0  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20220716 131042.554 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220716 131042.556 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.559 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.559 INFO             PET0  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20220716 131042.560 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.560 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.561 INFO             PET0  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20220716 131042.562 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.564 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.565 INFO             PET0  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20220716 131042.575 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.584 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.584 INFO             PET0  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20220716 131042.593 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.605 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.605 INFO             PET0  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20220716 131042.610 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.611 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.611 INFO             PET0  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20220716 131042.611 INFO             PET0  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20220716 131042.611 INFO             PET0 Finalizing ESMF
20220716 131042.554 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220716 131042.554 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220716 131042.554 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220716 131042.554 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220716 131042.554 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220716 131042.554 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220716 131042.554 INFO             PET1 Running with ESMF Version   : v8.4.0b04-2-gc831a7cd4a
20220716 131042.554 INFO             PET1 ESMF library build date/time: "Jul 16 2022" "12:50:27"
20220716 131042.554 INFO             PET1 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_9.1.0_openmpi_g_develop/esmf
20220716 131042.554 INFO             PET1 ESMF_COMM                   : openmpi
20220716 131042.554 INFO             PET1 ESMF_MOAB                   : enabled
20220716 131042.554 INFO             PET1 ESMF_LAPACK                 : enabled
20220716 131042.554 INFO             PET1 ESMF_NETCDF                 : enabled
20220716 131042.554 INFO             PET1 ESMF_PNETCDF                : disabled
20220716 131042.554 INFO             PET1 ESMF_PIO                    : enabled
20220716 131042.554 INFO             PET1 ESMF_YAMLCPP                : enabled
20220716 131042.554 INFO             PET1  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20220716 131042.554 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220716 131042.556 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.559 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.559 INFO             PET1  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20220716 131042.560 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.560 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.561 INFO             PET1  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20220716 131042.562 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.564 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.565 INFO             PET1  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20220716 131042.575 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.584 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.584 INFO             PET1  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20220716 131042.593 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.605 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.605 INFO             PET1  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20220716 131042.610 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.611 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.611 INFO             PET1  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20220716 131042.611 INFO             PET1  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20220716 131042.611 INFO             PET1 Finalizing ESMF
20220716 131042.551 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220716 131042.551 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220716 131042.551 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220716 131042.551 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220716 131042.551 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220716 131042.551 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220716 131042.551 INFO             PET2 Running with ESMF Version   : v8.4.0b04-2-gc831a7cd4a
20220716 131042.551 INFO             PET2 ESMF library build date/time: "Jul 16 2022" "12:50:27"
20220716 131042.551 INFO             PET2 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_9.1.0_openmpi_g_develop/esmf
20220716 131042.551 INFO             PET2 ESMF_COMM                   : openmpi
20220716 131042.552 INFO             PET2 ESMF_MOAB                   : enabled
20220716 131042.552 INFO             PET2 ESMF_LAPACK                 : enabled
20220716 131042.552 INFO             PET2 ESMF_NETCDF                 : enabled
20220716 131042.552 INFO             PET2 ESMF_PNETCDF                : disabled
20220716 131042.552 INFO             PET2 ESMF_PIO                    : enabled
20220716 131042.552 INFO             PET2 ESMF_YAMLCPP                : enabled
20220716 131042.554 INFO             PET2  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20220716 131042.554 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220716 131042.556 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.559 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.559 INFO             PET2  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20220716 131042.560 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.560 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.561 INFO             PET2  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20220716 131042.562 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.564 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.565 INFO             PET2  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20220716 131042.574 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.584 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.584 INFO             PET2  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20220716 131042.593 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.605 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.605 INFO             PET2  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20220716 131042.610 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.611 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.611 INFO             PET2  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20220716 131042.611 INFO             PET2  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20220716 131042.611 INFO             PET2 Finalizing ESMF
20220716 131042.553 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220716 131042.553 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220716 131042.553 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220716 131042.553 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220716 131042.553 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220716 131042.553 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220716 131042.553 INFO             PET3 Running with ESMF Version   : v8.4.0b04-2-gc831a7cd4a
20220716 131042.553 INFO             PET3 ESMF library build date/time: "Jul 16 2022" "12:50:27"
20220716 131042.553 INFO             PET3 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_9.1.0_openmpi_g_develop/esmf
20220716 131042.553 INFO             PET3 ESMF_COMM                   : openmpi
20220716 131042.553 INFO             PET3 ESMF_MOAB                   : enabled
20220716 131042.553 INFO             PET3 ESMF_LAPACK                 : enabled
20220716 131042.553 INFO             PET3 ESMF_NETCDF                 : enabled
20220716 131042.553 INFO             PET3 ESMF_PNETCDF                : disabled
20220716 131042.553 INFO             PET3 ESMF_PIO                    : enabled
20220716 131042.553 INFO             PET3 ESMF_YAMLCPP                : enabled
20220716 131042.554 INFO             PET3  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20220716 131042.554 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220716 131042.556 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.559 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.559 INFO             PET3  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20220716 131042.560 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.560 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.561 INFO             PET3  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20220716 131042.562 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.564 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.565 INFO             PET3  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20220716 131042.574 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.584 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.584 INFO             PET3  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20220716 131042.593 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.605 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.605 INFO             PET3  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20220716 131042.610 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220716 131042.611 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220716 131042.611 INFO             PET3  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20220716 131042.611 INFO             PET3  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20220716 131042.611 INFO             PET3 Finalizing ESMF
