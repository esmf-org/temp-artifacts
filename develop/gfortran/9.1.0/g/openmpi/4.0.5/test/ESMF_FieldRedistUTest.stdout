2022-07-15 14:13:46
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  PET           3  Test Elapsed Time    59.953999999999979       msec.
  PET           1  Test Elapsed Time    60.098000000000013       msec.
  PET           2  Test Elapsed Time    60.065999999999981       msec.
  PET           0  Test Elapsed Time    60.055000000000000       msec.
